content/en/serverless/step_functions/distributed-maps.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Tracing Distributed Map States
+---
+
+In AWS Step Functions, you can set up a large-scale parallel workload by including a [`Map` state in _Distributed_ mode][1]. Datadog supports tracing your Distributed Map states.
+
+{{< img src="serverless/step_functions/distributed_map.png" alt="A visualization of a Step Function with a Distributed Map state." style="width:100%;" >}}
+
+## Set up trace merging
+
+1. To ensure that child workflows are correctly linked to their parents, enable the **Item Batching** option for your Distributed Map state. For more information, see [ItemBatcher][2].
+
+2. Your State Machine definition must use [JSONata][4] as the query language. To enable this, set your definition's top-level `QueryLanguage` field to `JSONata`.
+
+3. On the Distributed Map state, set `_datadog` in the `BatchInput` field as follows:
+
+   {{< highlight json "hl_lines=4-4" >}}
+   "ItemBatcher": {
+     "MaxItemsPerBatch": 100,
```

Comment:
I think a note specifically calling this number out as user configurable would increase clarity

Reply:
Set the value to N to communicate that